Ginger/GingerCoreCommon/External/Configurations/VariableConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Amdocs.Ginger.Repository;
+
```

Comment:
what this class is for?

Reply:
I followed VRTCOnfigurations and this is what they have done there. I deleted this class and it works perfectly fine without it.